python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -184,6 +185,7 @@ def replace_illegal_columns_chars(char: str, database: BaseDatabase) -> str:
 
         return normalize_config
 
+    @typing.no_type_check
```

Comment:
Ignoring type check for this method for now since this is optional param in the operator because of that getting lot of error
```
src/astro/sql/operators/load_file.py: note: In member "get_openlineage_facets" of class "LoadFileOperator":
src/astro/sql/operators/load_file.py:228: error: Item "None" of
"Optional[BaseTable]" has no attribute "openlineage_emit_temp_table_event" 
[union-attr]
            if self.output_table.openlineage_emit_temp_table_event():
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:230: error: Item "None" of
"Optional[BaseTable]" has no attribute "openlineage_dataset_name"  [union-attr]
                    f"{self.output_table.openlineage_dataset_namespace()}"
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:230: error: Item "None" of
"Optional[BaseTable]" has no attribute "openlineage_dataset_namespace" 
[union-attr]
                    f"{self.output_table.openlineage_dataset_namespace()}"
                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:235: error: Item "None" of
"Optional[BaseTable]" has no attribute "openlineage_dataset_namespace" 
[union-attr]
                        namespace=self.output_table.openlineage_dataset_na...
                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~...
src/astro/sql/operators/load_file.py:2[36](https://github.com/astronomer/astro-sdk/actions/runs/3347578950/jobs/5545720608#step:6:37): error: Item "None" of
"Optional[BaseTable]" has no attribute "openlineage_dataset_name"  [union-attr]
                        name=self.output_table.openlineage_dataset_name(),
                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:2[39](https://github.com/astronomer/astro-sdk/actions/runs/3347578950/jobs/5545720608#step:6:40): error: Item "None" of
"Optional[BaseTable]" has no attribute "metadata"  [union-attr]
                                metadata=self.output_table.metadata,
                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:2[40](https://github.com/astronomer/astro-sdk/actions/runs/3347578950/jobs/5545720608#step:6:41): error: Item "None" of
"Optional[BaseTable]" has no attribute "columns"  [union-attr]
                                columns=self.output_table.columns,
                                        ^~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:2[41](https://github.com/astronomer/astro-sdk/actions/runs/3347578950/jobs/5545720608#step:6:42): error: Item "None" of
"Optional[BaseTable]" has no attribute "sqlalchemy_metadata"  [union-attr]
                                schema=self.output_table.sqlalchemy_metada...
                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:249: error: Item "None" of
"Optional[BaseTable]" has no attribute "metadata"  [union-attr]
                                        name=self.output_table.metadata.sc...
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:2[50](https://github.com/astronomer/astro-sdk/actions/runs/3347578950/jobs/5545720608#step:6:51): error: Item "None" of
"Optional[BaseTable]" has no attribute "metadata"  [union-attr]
                                        type=self.output_table.metadata.da...
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~
src/astro/sql/operators/load_file.py:2[54](https://github.com/astronomer/astro-sdk/actions/runs/3347578950/jobs/5545720608#step:6:55): error: Item "None" of
"Optional[BaseTable]" has no attribute "name"  [union-attr]
    ...       "dataSource": DataSourceDatasetFacet(name=self.output_table.nam...
                                                        ^~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Are we not getting this errors in other operators?

Reply:
I'm getting this because output_table is optional here

Reply:
@pankajastro I think we should resolve the typing issue before merging.

Reply:
@utkarsharma2 I have added an additional check to fix the above but not sure it is right idea or not